lib/PuppeteerSharp/ProcessException.cs

```diff
@@ -5,22 +5,22 @@ namespace PuppeteerSharp
     /// <summary>
     /// Chromium process exception thrown by <see cref="Launcher"/>.
     /// </summary>
-    public class ChromiumProcessException : PuppeteerException
```

Comment:
We should leave the class ChromiumProcessException and mark it as `[Obsolete("ProcessException will be thrown")]`